coderd/tracing/exporter.go

```diff
@@ -45,6 +48,17 @@ func TracerProvider(ctx context.Context, service string, opts TracerOpts) (*sdkt
 		closers = []func(context.Context) error{}
 	)
 
+	if opts.DataDog {
+		// See more:
+		// https://docs.datadoghq.com/tracing/metrics/runtime_metrics/go/
+		dd := ddotel.NewTracerProvider()
```

Comment:
I think you need something like this to actually enable runtime metrics, according to your link
```go
import ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"

dd := ddotel.NewTracerProvider(ddtracer.WithRuntimeMetrics())
```